src/components/login/AuthDetails.tsx

```diff
@@ -0,0 +1,28 @@
+import { onAuthStateChanged, signOut } from "firebase/auth";
+import { useEffect, useState } from "react";
+import { auth } from '../../firebase-config';
+
+const AuthDetails = () => {
+    const [authUser, setAuthUser] = useState(null);
+
+    useEffect(() => {
+        const listen = onAuthStateChanged(auth, (user) => {
```

Comment:
Recommend renaming 'listen' to 'removeAuthListener' since that more clearly communicates that this funciton will remove the listener that was setup when a subscription was created that runs a callback whenver the authentication state changes. 
For example:
`const listen = onAuthStateChanged(auth, (user) => {`

Reply:
done. Is this more so just have better naming conventions as to make it more clean code for it to be understood by the next person?